### PR TITLE
process doc imporvement:  input sessions flow

### PR DESCRIPTION
**Pull request by tamari-gray**
based on my findings from creating input sessions this week, hopefully it deepens the flow of the process (doc)!

### CODE REVIEW

**Review by nickmeinhold — APPROVED** [review]
Love it!  Thanks for these awesome improvements!!!

File: .contributing/the-process.md [hunk 1]
```diff
@@ -167,15 +167,15 @@ Key
...
+        - Add a link to the google drawing in the task description
+        - Move the task to the "Open for input" column
+        - Add a due date of tomorrow at 5:00pm
+        
+    2. Hold an RTC to discuss the file. This is to get all teammates on the same page, which makes it easier for everyone to give more valuable input. 
+        - RTC can be held during day of creation, or after standup next day.
+        - At end of RTC, inform team about input sessions due date 
+
+    3. Finalise file:
+        - Resolve any conversations (use RTC's if neccessary)
```
Comment (T-Davis) [inline .contributing/the-process.md:180]:
```suggestion
        - Resolve any conversations (use RTCs if necessary)
```

File: .contributing/the-process.md [hunk 2]
```diff
@@ -167,15 +167,15 @@ Key
 2. "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* < User Flow Pieces < *Section* (CL)"
 
 3. "input task": 
-    - e.g Create a task in Asana with title "IS [1]: *title of Google file*" and:
-      - Add a link to the google drawing in the task description
```
Comment (T-Davis) [inline .contributing/the-process.md:171]: Maybe Google Doc instead? We could be getting input on use cases

**Review by T-Davis — CHANGES_REQUESTED** [review]
Hey good work Tam, just one comment to have you check out

File: .contributing/the-process.md (hunk 2 above)
Comment (T-Davis) [reply .contributing/the-process.md:171]: This is supposed to reference the new part of the file that says google drawing

Comment (tamari-gray) [reply .contributing/the-process.md:171]: yess!!

**Review by T-Davis — APPROVED** [review]
Awesome work as always 🙏

**Review by nickmeinhold — APPROVED** [review]
Looks great!  Awesome work mate!!!

**Pull request merged**